### PR TITLE
chor(package.json) until https://github.com/rollup/rollup/pull/4443#issuecomment-1067011744 can get merged

### PR DESCRIPTION
**Pull request by frank-dspeed**
chor until https://github.com/rollup/rollup/pull/4443#issuecomment-1067011744 can get merged

## Description

This allows the most possible compat between nodejs versions as also require import usage with current nodejs versions

### CODE REVIEW

**Comment by frank-dspeed** [comment]
@lukastaegert @guybedford please review i think that my conclusion is correct and thats the most compatible version at present that allows at last extension less imports when require is used.

File: package.json [hunk 1]
```diff
@@ -128,14 +128,9 @@
...
       "types": "./dist/rollup.d.ts",
-      "node": {
-        "require": "./dist/rollup.js",
-        "import": "./dist/es/rollup.js"
-      },
+      "require": "./dist/rollup.js",
+      "import": "./dist/es/rollup.js",
       "default": "./dist/es/rollup.browser.js"
-    },
-    "./dist/*": "./dist/*"
```
Comment (aladdin-add) [inline package.json:139]: it indeed is a breaking change to remove this - as import `rollup/dist/xxx` will throw an error. :)

**Comment by lukastaegert** [comment]
This cannot be merged as it prevents https://rollupjs.org/guide/en/#programmatically-loading-a-config-file which is part of the public API.

**Comment by frank-dspeed** [comment]
## Workaround
@lukastaegert your correct but some one pointed the most best trick out in a other issue here which was related he gave me the tip to simply disable all npm scripts this enables to clone rollup into a subfolder and that root folder where it gets cloned it can be used as build config to build rollup clean as needed.

I even got the final conclusion now that it makes zero sense to even offer a nodejs rollup esm build i drop that complet from my stack as the nodejs rollup version is bound to all NodeJS Internal Restrictions like no Real ESM Entrypoint without a CJS Entrypoint i can keep it cjs and reuse it with clean named exports in both environments. And as i got a wrapper package that cleans the package.json stuff all will work forever finaly no matter what the package.json of rollup says.

**Comment by lukastaegert** [comment]
Considering all options, I would prefer to just leave it as it is for now. The only downside is that everyone is forced to use extensions, beyond that I think we should be fine.

**Pull request closed without merging**